src/CreatorToken.sol

```diff
@@ -64,6 +111,35 @@ contract CreatorToken is ERC721 {
     payToken.safeTransferFrom(msg.sender, admin, _adminFee);
   }
 
+  function _sell(uint256 _tokenId, uint256 _minAcceptedPrice) internal whenNotPaused {
+    if (msg.sender != ownerOf(_tokenId)) {
+      revert CreatorToken__CallerIsNotOwner(_tokenId, ownerOf(_tokenId), msg.sender);
+    }
+    if (circulatingSupply == 1) revert CreatorToken__LastTokenCannotBeSold(circulatingSupply);
+
+    uint256 _tokenPrice = _temporaryGetNextTokenPrice();
+    (uint256 _creatorFee, uint256 _adminFee) = calculateFees(_tokenPrice);
+    uint256 _totalPrice = _tokenPrice - _creatorFee - _adminFee;
+
+    if (_totalPrice < _minAcceptedPrice) {
+      revert CreatorToken__MinAcceptedPriceExceeded(_totalPrice, _minAcceptedPrice);
+    }
+
+    transferFrom(msg.sender, address(this), _tokenId);
+    _burn(_tokenId);
```

Comment:
It looks like we don't decrement the circulating supply when we burn this. We should probably pull it into a small internal helper method `_burnAndDecrement` and do that. In addition to adding this, we should make sure the sell tests catch it. Thanks!

Reply:
nice catch, fixed here: https://github.com/showtime-xyz/creator-tokens/pull/13/commits/3ad5836cd07b5ec29ed6f04b6058fde8a08e27a2